src/Resources/public/Admin.js

```diff
@@ -226,6 +226,10 @@ var Admin = {
                     .replaceWith(html);
             },
             error: function(xhr, statusText, errorThrown) {
+                if (xhr.status === 400 || xhr.status === 403 || xhr.status === 404 || xhr.status === 405){
```

Comment:
Couldn't we trust the content-type header instead?

Reply:
That would be great to have a better check than status code indeed.

Reply:
No news from @leoBergerot, do you know how to update the PR in order to use the content-type header @phansys ?

Reply:
See #6856.